docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-download-proc.adoc

```diff
@@ -0,0 +1,20 @@
+[id='optimizer-download-proc']
+= Download {PLANNER}
+
+{PLANNER} is production ready.
+The API is stable but backward incompatible changes can occur.
+
+.Prerequisite
+Download and install {PRODUCT} from the https://access.redhat.com[Red Hat Customer Portal]. See the {INSTALLATION_GUIDE} for more information about installing {PRODUCT}.
+
+.Procedure
+. Download the {PLANNER} file from the https://access.redhat.com[Red Hat Customer Portal].
+. Unzip the {PLANNER} files.
+
+[float]
+== Upgrading from previous versions
```

Comment:
This could be rather a part of migration guide, what do you think?

Reply:
I've commented this out for now and passed the information on to Emily Murphy, who is writing the migration user stories for this release.